graylog2-server/src/main/java/org/graylog2/contentpacks/ContentPackable.java

```diff
@@ -24,4 +25,9 @@
     default void resolveNativeEntity(EntityDescriptor entityDescriptor,
                                      MutableGraph<EntityDescriptor> mutableGraph) {
     }
+
+    @JsonIgnore
+    default String getPackageName() {
```

Comment:
What do you think about using a more content pack specific name for this? As it its now, the method name is pretty generic. How about `getContentPackPluginPackage()` or something similar?